src/raven.js

```diff
@@ -413,6 +413,16 @@ Raven.prototype = {
             timestamp: now() / 1000
         }, obj);
 
+        if (isFunction(this._globalOptions.breadcrumbCallback)) {
+            var result = this._globalOptions.breadcrumbCallback(crumb);
+
+            if (result) {
+                crumb = result;
+            } else {
```

Comment:
> Is this the behavior we want for the breadcrumbCallback?

No. We're going to update how `dataCallback` works in a follow-up. Basically, your work here on `breadcrumbCallback` will be the model for how callbacks should work – presuming we all feel confident about it.